ExternalModule.php

```diff
@@ -119,7 +119,10 @@ function redcap_every_page_top($project_id) {
      */
     protected function includeJs($path) {
         // Use noauth method, but not the API endpoint, to load resources while not in network
-        $ext_path = $this->getUrl($path, true, false);
+        // $ext_path = $this->getUrl($path, true, false);
+        $use_api_urls = $this->getSystemSetting('use-api-urls');
```

Comment:
```suggestion
        $use_api_urls = ($this->getSystemSetting('use-api-urls')) ?: false; 
```
Not sure if `null`'s falsiness is actually an issue, bit I'm leaving this suggestion here just in case.

Reply:
fine either way I think, but not harm.